docs/templates/snakemake/rulegraph.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+snakemake \
+    --configfile "config/config.[METHOD].yaml" \
```

Comment:
Better to add `--snakefile` as in `dryrun.sh` and `run_local.sh` for consistency

Reply:
done